src/doc/data_into_visit/data_examples/xdmf.C

```diff
@@ -0,0 +1,2331 @@
+// Copyright (c) Lawrence Livermore National Security, LLC and other VisIt
+// Project developers.  See the top-level LICENSE file for dates and other
+// details.  No copyright assignment is required to contribute to VisIt.
+
+#include <stdio.h>
+#include <stdlib.h>
+
+// Define this symbol BEFORE including hdf5.h to indicate the HDF5 code
+// in this file uses version 1.6 of the HDF5 API. This is harmless for
+// versions of HDF5 before 1.8 and ensures correct compilation with
+// version 1.8 and thereafter. When, and if, the HDF5 code in this file
+// is explicitly upgraded to the 1.8 API, this symbol should be removed.
+#define H5_USE_16_API
```

Comment:
Question: is this file really using 1.6 API? Can it be updated to 1.8?  Just wondering why the introduction of a file using the old api?